### ISSUE TITLE
Please tag a version so we can use in composer

### ISSUE THREAD

**Issue by timwhite** [kravock/php-smsbroadcast#1]
Currently there isn't a "stable" release with a tag, which makes it harder to just ```composer require kravock/php-smsbroadcast```

It would be great to have it tagged

**Comment by kravock** [comment]
Hey mate, sorry about that. I will check that everything is stable and set that up.

**Comment by kravock** [comment]
Hey, should be all good to go now.

**Comment by timwhite** [comment]
I also made some minor changes to the library (https://github.com/timwhite/php-smsbroadcast). Made it PSR-4 autoloadable as I had issues when it was just PSR-0. And then ensured we urlencode where required. Otherwise numbers in the format of +61 cause an error as the + can be interpreted as a space. According to the API docs, there a number that should be urlencoded, so I've fixed them in my branch.
Given the PSR-0 to PSR-4 change, I won't make a pull request unless you are happy to make that change.

Thanks for the library!
